webpack.config.js

```diff
@@ -12,12 +16,15 @@ module.exports = {
     loaders: [
       { test: /\.js?$/, loaders: ['react-hot', 'babel'], exclude: /node_modules/ },
       { test: /\.js$/, exclude: /node_modules/, loader: 'babel-loader'},
-      { test: /\.scss$/, loader: "style!css!sass" }    
+      { test: /\.scss$/, loader: "style!css!postcss-loader!sass" }    
 
     ]
   },
   plugins: [
     new webpack.NoErrorsPlugin()
-  ]
+  ],
+  postcss: function () {
+    return [autoprefixer, csswring]
```

Comment:
auto-adds vendor prefixes to some css rules (-moz, -webkit, etc), after the css has been generated from scss

we're using a "transition" css rule that needs this to work in all browsers, and we will use others as time goes by